.github/workflows/release.yml

```diff
@@ -30,19 +30,63 @@ jobs:
           args: --release
       - name: Prepare release files
         run: |
-          mv target/release/drml .
-          mv target/release/wbuild/pangolin-runtime/*.compressed.wasm .
-          mv target/release/wbuild/pangoro-runtime/*.compressed.wasm .
+          mkdir artifacts
+          mv target/release/drml artifacts
+          mv target/release/wbuild/pangolin-runtime/*.compressed.wasm artifacts
+          mv target/release/wbuild/pangoro-runtime/*.compressed.wasm artifacts
           tar cf drml.tar.zst drml -I pzstd
+          mv drml.tar.zst artifacts
+      - name: Upload
+        uses: actions/upload-artifact@v2.2.4
+        with:
+          name: artifacts
+          path: artifacts
+
+  build-with-evm-tracing:
+    name: Build with evm-tracing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch latest code
+        uses: actions/checkout@v3
+      - name: Install Rust nightly toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ env.RUST_TOOLCHAIN }}
+          default: true
+      - name: Build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --features evm-tracing
+      - name: Prepare release files
+        run: |
+          mkdir artifacts
+          mv target/release/drml artifacts
+          tar cf drml-tracing-node.tar.zst drml -I pzstd
+          mv drml-tracing-node.tar.zst artifacts
+      - name: Upload
+        uses: actions/upload-artifact@v2.2.4
+        with:
+          name: artifacts
+          path: artifacts
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
```

Comment:
Missing `needs` for this job.

```
needs: [build, build-with-evm-tracing]
```

Reply:
Damn.
I have thought about this at the beginning.
But I forgot that at the end.